GoogleForCreators/web-stories-wp
# Add re-trimming from original video

## Context



## Summary
Adds re-trimming for videos so that the video is always trimmed from the original video instead of the current video.


## Relevant Technical Choices



## To-do

- [x] Make sure the original video is actually used for trimming as well



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Add a video element, longer (e.g. 30 seconds) is better for testing.
2. Trim it very short (e.g. 2 seconds)
3. Now enter trim mode again. Verify that the video trim menu still shows the full length of the video and the "active" part is accurate length trimmed in the previous step.
4. Now trim the video longer again (e.g. 15 sec)
5. Verify it works as expected, a new video with accurate length is created, and when you open the trim mode again, the video trim menu shows the full length again.


### Does this PR change what data or activity we track or use?



## Checklist




---



Fixes #8892
